README.md

```diff
@@ -175,30 +175,40 @@ expect(1..10).to cover(3)
 ### Collection membership
 
 ```ruby
-expect(actual).to include(expected)
+# exact order, entire range
+expect(actual).to eq(expected)
+
+# exact order, partial range (based on an exact position)
 expect(actual).to start_with(expected)
 expect(actual).to end_with(expected)
 
-expect(actual).to contain_exactly(individual, items)
-# ...which is the same as:
-expect(actual).to match_array(expected_array)
+# any order, entire range
+expect(actual).to match_array([expected_element1, expected_element2])
+# Assuming that:
+#   expected = [expected_element1, expected_element2]
+# You can also express this in a variadic function form:
```

Comment:
I've never heard the term "variadic function form".  How about "You can also express this by passing the expected elements as individual arguments:"